Clarify hook declaration (#192)
projectriff/node-function-invoker: README.md

```

The lifecycle methods are optional, and should only be implemented when needed.

> Note that they must necessarily be exported **after** the function (any hook exported before would be overwritten by the function export).

Reviewer: after implies a constraint that isn't true. The important aspect is that the lifecycle methods are on the function, not the export object. The export object is replaced by the function.

These are both correct

```js
const echo = msg => msg;
echo.$init = () => console.log('init');
module.exports = echo;
```

```js
echo.$init = () => console.log('init');
module.exports = echo;
function echo(msg) { return msg; }
```

Author: You are right. However, I cannot find a good wording for this that will be correct and not confuse users at the same time. Dropping the PR for now.

Reviewer: ```suggestion
Note that the lifecycle hooks must be fields on the exported function.
```
